plutus-benchmark/marlowe/src/PlutusBenchmark/Marlowe/BenchUtil.hs

```diff
@@ -282,7 +282,7 @@ executeBenchmark serialisedValidator Benchmark{..} =
               [bDatum, bRedeemer, toData bScriptContext]
 
 -- | The execution context for benchmarking.
-evaluationContext :: Either String EvaluationContext
+evaluationContext :: Either String (EvaluationContext PLC.DefaultFun)
```

Comment:
If we're gonna merge this, we should probably introduce a synonym for `EvaltuationContext DefaultFun`.